packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -988,9 +992,18 @@ export default {
                 error: ANALYTICS_ERROR_CODE_INVALID_MODEL_STATE
             });
```

Comment:
Should we remove the invalid model state log as well @SachaZvetelman ?

Reply:
That's a really good question. I'll need to confirm with Jack. Maybe leave it for the time being and we can remove it later so this PR doesn't get delayed.

Reply:
Yep, I just mentioned it as it was visible here! I'll take a note to remember to follow up.

Reply:
I'd rather keep the PR focused, I'm quite happy to create a new one though is there's tidy-up outside of this.